midpoint/install/system-requirements/index.adoc

```diff
@@ -190,135 +196,133 @@ This is important to keep in mind when choosing the right HA strategy.
-The database engine installation is dedicated to midPoint.
+MidPoint does not explicitly support repository-level clustering.
+Even in big deployments, the bottle neck is usually the amount of resources or the effective communication speed of the resources.
 
-This is the most expensive setup and is seldom justifiable due to the cost of the HA database system.
-The usual compromise in this case is to use link:https://www.geeksforgeeks.org/system-design/active-passive-active-active-architecture-for-high-availability-system/[active-passive] database HA strategies. 
-Due to the low criticality of midPoint, this is usually acceptable from the operational point of view.
+We suggest to primarily *set up a proper backup solution*, verify the time to recovery, and always xref:/midpoint/reference/simulation/[*simulate everything* before you make changes to production].
 
-== Software Requirements
+If you require clustering the DB, you can set up an link:https://www.geeksforgeeks.org/system-design/active-passive-active-active-architecture-for-high-availability-system/[active-passive] failover mechanism and take advantage of the PostgreSQL database-side clustering support on the JDBC driver level.
 
-Refer to the xref:/midpoint/release/[midPoint Releases] documentation for software requirements.
+// This is best covered in [midPoint] Advice on Database High Availability --  https://lists.evolveum.com/pipermail/midpoint/2024-May/008167.html
 
-== Infrastructure Requirements
+== Software and Infrastructure Requirements
 
 When starting an IAM project, you must prepare not only the midPoint servers but also the database and load balancer (if required).
 You need access to the infrastructure where these servers are running, as well as to the source and target systems.
-In most cases, the infrastructure is prepared by the administrators on the customer site.
+
+Refer to the xref:/midpoint/release/[midPoint Releases] documentation for software requirements.
 
 === Basic Single-Node Deployment
 
-The following schema represents a basic scenario:
+The following schema represents a basic deployment environment:
 
-//image::environment-schema-basic.png[]
-image::midpoint-environment-schema-basic.svg[title="Basic single-node midPoint deployment"]
+.Basic single-node midPoint deployment
+image::midpoint-environment-schema-basic.svg["Basic midPoint deployment schema with midPoint and its repository in the center; lines showing access routes to example source and target resources and user's computer. Optional VPN is placed between midPoint and the computer"]
 
 In the center of the schema, the largest rectangle represents a virtual machine, usually running Linux, with the xref:/midpoint/install/bare-installation/distribution/#purpose-and-quality[basic required set of tools] installed.
+MidPoint runs on this virtual machine.
+The database repository preferably runs on a separate virtual machine, albeit possibly on the same physical server.
```

Comment:
I know containers are mentioned later, but this basic schema is valid for containers too (omitting container specifics). Can we mention containers here too?